src/js/disability-benefits/components/ClaimsListItem.jsx

```diff
@@ -1,17 +1,26 @@
 import React from 'react';
 import { Link } from 'react-router';
+import moment from 'moment';
+
+import { getPhaseDescription } from '../utils/helpers';
 
 export default function ClaimsListItem({ claim }) {
-  const filesNeeded = claim.attributes.trackedItems.stillNeedFromYouList ? claim.attributes.trackedItems.stillNeedFromYouList.length : null;
   return (
     <Link className="claim-list-item" to={`/your-claims/${claim.id}/status`}>
       <h4 className="claim-list-item-header">Compensation Claim</h4>
-      <p className="status">Status: {claim.attributes.phaseDates.latestPhaseType}</p>
-      {filesNeeded !== null
-        ? <p><i className="fa fa-exclamation-triangle"></i>We need {filesNeeded} {filesNeeded > 1 ? 'files' : 'file'} from you</p>
-        : null}
-      <p><i className="fa fa-envelope"></i>We sent you a development letter (TODO)</p>
-      <p>Last Update: TODO</p>
+      <p className="status">Status: {getPhaseDescription(claim.attributes.phase)}</p>
+      <div className="communications">
+        {claim.attributes.developmentLetterSent && !claim.attributes.decisionLetterSent
+          ? <p><i className="fa fa-envelope"></i>We sent you a development letter</p>
+          : null}
+        {claim.attributes.decisionLetterSent
+          ? <p><i className="fa fa-envelope"></i>We sent you a decision letter</p>
+          : null}
+        {!claim.attributes.decisionLetterSent && claim.attributes.documentsNeeded
+          ? <p><i className="fa fa-exclamation-triangle"></i>We need files from you</p>
+          : null}
+      </div>
+      <p>Last update: {moment(claim.attributes.updatedAt).format('MMM M, YYYY')}</p>
```

Comment:
Ah, I didn't notice that. The date I was using before was `phaseChangeDate`, which is the only other update type date I see.